### DISCUSSION TITLE
User based views & filters [May 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2167] [Discord Threads]
> 🙋‍♂️ **Martin**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@403154219256905728> Currently NocoDB does not support User groups,  'CreatedBy' and 'ModifiedBy' column type's. Access control provided is at role-level (creator, editor, commenter, viewer).  
#  
> 🙋‍♂️ **Martin**  
#### Thanks for the super quick answer. So for now it's limited to role+table, cprrect? No way to "filter" what a given user can see in a specific table  
#  
> 👨‍🦲 **dstala**  
#### We can create shared read only views with required filters applied & share link across to specific set of people (optionally with password protection enabled).   
####    
#### But for people part of the project (collaborators) - its limited to role based model.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976523107160444959)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
